pom.xml

```diff
@@ -235,17 +235,17 @@
 			<dependency>
 				<groupId>org.hibernate</groupId>
 				<artifactId>hibernate-core</artifactId>
-				<version>${hibernateVersion}</version>
+				<version>6.0.0.Alpha2</version>
 			</dependency>
 			<dependency>
 				<groupId>org.hibernate</groupId>
 				<artifactId>hibernate-c3p0</artifactId>
-				<version>${hibernateVersion}</version>
```

Comment:
when i look at this,it looks like you are overriding the hibernate  version defined under properties in one of your poms